prow/jobs/kyma/kyma-integration.yaml

```diff
@@ -952,6 +944,29 @@ periodics:
       path_alias: github.com/kyma-project/test-infra
     - <<: *kyma_ref
       base_ref: master
+  spec:
```

Comment:
There is already a spec for this job added by `gke_backup_job_template`. Please delete this one and add the env var there.

Reply:
thanks, it is done.